apps/src/templates/studioHomepages/SetUpMessage.jsx

```diff
@@ -74,11 +59,11 @@ const SetUpMessage = ({
   onClick,
 }) => (
   <div style={styles.section} className={className}>
-    <div style={styles.wordBox}>
-      <div style={isRtl ? styles.rtlHeading : styles.heading}>
+    <div style={[styles.wordBox, isRtl ? styles.rtl : styles.ltr, {padding:0}]}>
```

Comment:
Also feels strange to have ltr/rtl define some padding, only to override it immediately thereafter. Also, Radium will often complain about mixed methods for padding (i.e. using both the specific `paddingLeft` and the general `padding`)